docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,344 @@ redirect_from:
+
+<!--TODO - Update image to match the latest App Directory sidebar (but remove the Workflows beta) -->
+![OAuth Tokens](../assets/bot-token.png "Bot OAuth Token")
+
+> 💡 Treat your token like a password and [keep it safe](https://api.slack.com/docs/oauth-safety). Your app uses it to post and retrieve information from Slack workspaces.
+
+---
+
+### Setting up your local project
+With the initial configuration handled, it's time to set up a new Bolt project. This is where you'll write the code that handles the logic for your app.
+
+If you don’t already have a project, let’s create a new one. Create an empty directory:
+
+```shell
+mkdir first-bolt-app
+cd first-bolt-app
+```
+
+Next, we recommend using a Python virtual environment to manage your project's dependencies. Create and activate a new virtual environment:
+
+<details>
+<summary markdown="0">
+<h4>Setting up a Python virtual environment</h4>
+</summary>
+
+TODO - This is where we explain how to installing a Python version manager, virtual environment, and adding it to gitignore.
+
+---
+</details>
+
+<!-- TODO - Details on installing Python version manager, virtual environment, adding it to the git ignore, etc -->
+<!-- TODO - Do we use python or python3? -->
+<!-- TODO - Should we confirm that the virtual environment is active? -->
+```shell
+python3 -m venv env
+source env/bin/activate
+```
+
+Before we install the Bolt for Python package to your new project, let's save the bot token and signing secret that was generated when you configured your app. These should be stored as environment variables and should *not* be saved in version control.
+
+1. **Copy your Signing Secret from the Basic Information page** and then store it in a new environment variable. The following example works on Linux and MacOS; but [similar commands are available on Windows](https://superuser.com/questions/212150/how-to-set-env-variable-in-windows-cmd-line/212153#212153).
+
+```shell
+export SLACK_SIGNING_SECRET=<your-signing-secret>
+```
+
+<!-- TODO - Fix the bullet displaying 1. -->
+2. **Copy your bot (xoxb) token from the OAuth & Permissions page** and store it in another environment variable.
+
+```shell
+export SLACK_BOT_TOKEN=xoxb-<your-bot-token>
+```
+
+<!-- TODO - Is `slack_bolt` called a package in Python? -->
+Now, lets create your app. Install the `slack_bolt` package to your virtual environment using the following command:
+
+```shell
+pip install slack_bolt
+```
+<!-- TODO - Do we want to mention that it will be installed to env/.../site-packages? -->
+
+Create a new file called `app.py` in this directory and add the following code:
+
+<!-- TODO - Is it best practice to separate imports and froms with a new line? -->
+<!-- TODO - Is it best practice to have 2 new lines after the imports? -->
+<!--
+  TODO - Should the code sample include the Logger middleware that's common in many of the samples?
```

Comment:
In Bolt JS, it is part of advanced topics: https://slack.dev/bolt-js/concepts#global-middleware

Reply:
Perfect. I'll leave the logger middleware out of the Getting Started guide and we can explain it in the advanced concepts.